css/resched_options.css

```diff
@@ -1,15 +1,23 @@
-h2,div { 
-	width: 80%;
-	margin: 20px 10%; 
+h2,div {
+	margin: 0 10% 20px 10%;
 }
-a { 
+a {
 	text-decoration: none;
 	color: black;
 }
-.option { 
-	height: 70px;
-	line-height: 70px;
-	border: 1px solid black;
-	text-align: center; 
+.option {
+	padding: 35px;
+	border: 1px solid #eee;
+	border-radius: 2px;
 	cursor: pointer;
-}
\ No newline at end of file
+	background-color: white;
+	text-align: left;
+}
+.option p:first-of-type {
```

Comment:
Leaves behind IE8, doesn't it?

(Not sure we should care. We probably shouldn't.)